lxd/instance/drivers/driver_qemu.go

```diff
@@ -784,12 +785,27 @@ func (d *qemu) Rebuild(img *api.Image, op *operations.Operation) error {
 	return d.rebuildCommon(d, img, op)
 }
 
-func (d *qemu) ovmfPath() string {
-	if os.Getenv("LXD_OVMF_PATH") != "" {
-		return os.Getenv("LXD_OVMF_PATH")
+func (*qemu) fwPath(filename string) string {
+	qemuFwPaths := "/usr/share/OVMF"
+	if os.Getenv("LXD_QEMU_FW_PATH") != "" {
+		qemuFwPaths = os.Getenv("LXD_QEMU_FW_PATH")
+	} else if os.Getenv("LXD_OVMF_PATH") != "" {
+		qemuFwPaths = os.Getenv("LXD_OVMF_PATH")
 	}
 
-	return "/usr/share/OVMF"
+	for _, path := range strings.Split(qemuFwPaths, ":") {
```

Comment:
We could use the `qemuFwPaths()` function described above here too.

Reply:
Have done.